client/app/src/test/java/com/example/criengine/BookUnitTest.java

```diff
@@ -0,0 +1,176 @@
+package com.example.criengine;
+
+import com.example.criengine.Objects.Book;
+import org.junit.Test;
+import java.util.ArrayList;
+import java.util.List;
+import static org.junit.Assert.*;
+
+/**
+ * Unit tests for the Book.java class.
+ * These tests test every method in the class.
+ */
+public class BookUnitTest {
+    @Test
+    public void testBookConstructorOne() {
+        Book newBook = new Book();
+
+        assertEquals(newBook.getRequesters().size(), 0);
+    }
+
+    @Test
+    public void testBookConstructorTwo() {
+        List<String> newRequesterList = new ArrayList<>();
+        Book newBook = new Book("ID 1", "New Owner", "New Title", "New Author", "New Description", "New ISBN", "New Status", "New Borrower", newRequesterList, "New Geolocation", "New Image");
```

Comment:
for all of these, you can do a @Before to generate them

Reply:
will do!